symfony/plugins/orangehrmCorePlugin/Service/DateTimeHelperService.php

```diff
@@ -131,4 +131,53 @@ public function getNow(DateTimeZone $timezone = null): DateTime
     {
         return new DateTime('now', $timezone);
     }
+
+    /**
+     * @param $baseTimestamp
+     * @return DateTime
+     */
+    public function getDateTimeByTimestamp($baseTimestamp): DateTime
+    {
+        $dateTime = new DateTime();
+        return $dateTime->setTimestamp($baseTimestamp);
+    }
```

Comment:
what is the type of `$baseTimestamp`. `$dateTime = new DateTime();` doing this we can't mock in unit tests. better to use `$this->getNow()`.
Can't we handle this with `public function getNow(DateTimeZone $timezone = null): DateTime` since it allows timezone.

Reply:
refactored